library/Icinga/Web/StyleSheet.php

```diff
@@ -149,6 +149,20 @@ protected function collect()
                 $this->lessCompiler->setTheme($this->pubPath . '/css/themes/' . $theme . '.less');
             }
         }
+
+        if ($user = Auth::getInstance()->getUser()) {
+            $mode = $user->getPreferences()->getValue('icingaweb', 'mode');
+
+            if (($theme === null || $theme === self::DEFAULT_THEME) && empty($mode)) {
+                $mode = 'dark';
+            } elseif (empty($mode)) {
+                $mode = 'none';
+            }
+
+            $this->lessCompiler->addLessFile($this->pubPath . '/css/modes/'. $mode .'.less');
+        } else {
+            $this->lessCompiler->addLessFile($this->pubPath . '/css/modes/none.less');
```

Comment:
We should introduce a constant like `self::DEFAULT_THEME` analogous to the the mode. So please add `self::DEFAULT_MODE` and use it.

Also, don't apply the mode `none` for anonymous users, apply the default mode.